react/components/EXPERIMENTAL_Table/SimpleTable/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FC, memo } from 'react'
+
+import Cell from './Cell'
+import useTableContext from '../hooks/useTableContext'
+import constants from '../constants'
+
+const SimpleTable: FC = () => {
+  const { columns, items } = useTableContext()
+  const rowClassNames = 'dt-row w-100 h-100 ph4 truncate overflow-x-hidden'
+
+  const renderHeader = (headerData: string, headerIndex: number) => {
+    const headerRender = columns[headerData].headerRender
+    const content = headerRender
+      ? headerRender({ headerData })
+      : columns[headerData].title
+    return <Cell key={`col-${headerIndex}`} content={content} isHeader />
+  }
+
+  const renderRow = (rowData: Object, rowIndex: number) => {
+    return (
+      <div
+        style={{ height: constants.ROW_HEIGHT }}
+        className={rowClassNames}
+        key={`row-${rowIndex}`}>
+        {Object.keys(rowData).map((cel: string, cellIndex: number) => {
+          const cellRender = columns[cel].cellRender
+          const cellData = rowData[cel]
+          const content = cellRender
+            ? cellRender({ cellData, rowData })
+            : cellData
+          return <Cell key={`${rowIndex}-${cellIndex}`} content={content} />
+        })}
+      </div>
+    )
+  }
+
+  const header = (
```

Comment:
Just a thought, maybe this should be a component, with renderHeader too and getting columns as props...

Reply:
Great suggestion, just did that (with the body as well)! PS. There is no need to pass as a prop since we can consume it via context   🤓